jina/clients/base/grpc.py

```diff
@@ -84,58 +109,65 @@ async def _get_results(
                 with ProgressBar(
                     total_length=self._inputs_length, disable=not self.show_progress
                 ) as p_bar:
-                    async for resp in stub.Call(
-                        req_iter,
-                        compression=self.compression,
-                        metadata=kwargs.get('metadata', None),
-                        credentials=kwargs.get('credentials', None),
-                        timeout=kwargs.get('timeout', None),
-                    ):
-                        callback_exec(
-                            response=resp,
-                            on_error=on_error,
-                            on_done=on_done,
-                            on_always=on_always,
-                            continue_on_error=self.continue_on_error,
-                            logger=self.logger,
-                        )
-                        if self.show_progress:
-                            p_bar.update()
-                        yield resp
+                    try:
+                        async for resp in stub.Call(
```

Comment:
Should we make the retry here? As I understand this code,it will still fail in case of a network error. Right?
So what about the case where the wifi is down for a few seconds? Will it retry?

Reply:
the `grpc` channel retry options should be the one handling this.